mlir/utils/performance/analysis/compileAndCollectTuningData.py

```diff
@@ -0,0 +1,604 @@
+"""
+This script compiles a given config in order to collect the following data
+points for each config:
+- Blocksize
+- Gridsize
+- vgpr
+- sgpr
+- LDS allocated
+- Occupancy
+- WFsPerWG
+- mfma_wmma_instruction
+
+The given config is expected to be a tsv with the following format:
+|# arch| numCUs | testVector | perfConfig (exhaustive) |
+
+Usage:
+    python3 compileAndCollectTuningData.py --op <operation> <config.tsv>
+"""
+
+import argparse
+import csv
+import os
+import perfRunner
```

Comment:
If it happens that the script is not ran from Jenkins we would have ModuleNotFoundError for perfRunner, when running the script using `python3 compileAndCollectTuningData.py --op <operation> <config.tsv>`. Is there a way to import perfRunner from `mlir/utils/performance/perfRunner.py`, if ci-performance-scripts is not executed. (this would make sense only if this script is intended to be used outside of CI, for example in docker we use for development).
Maybe smth like:
```suggestion
try:
    import perfRunner
except ModuleNotFoundError:
    parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
    sys.path.append(parent_dir)
    import perfRunner
```